frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylist.tsx

```diff
@@ -23,6 +23,70 @@ import {
 import { SessionRecordingsPlaylistSettings } from './SessionRecordingsPlaylistSettings'
 import { SessionRecordingsPlaylistTroubleshooting } from './SessionRecordingsPlaylistTroubleshooting'
 
+function SortedBy({
+    filters,
+    setFilters,
+}: {
+    filters?: RecordingUniversalFilters
+    setFilters?: (filters: Partial<RecordingUniversalFilters>) => void
+}): JSX.Element | null {
+    return filters && setFilters ? (
```

Comment:
Personal preference but I typically do this type of checking outside the component and make the props mandatory. It can be confusing when reading code that implements this component to understand why it isn't rendering without looking at its implementation otherwise

Reply:
yeah, i go backwards and forwards on this.

i feel like a lot of react components are hard to read because we mix presentation and arrangement

i wish sometimes we had things like

```
<replay scene>
 <filters/>
 <playlist/>
 <player/>
 <inspector/>
</replay scene>
```

but totally agree that means you have to read every component

and in this case agree it does feel like passing my trousers to the shopkeeper so they can find my wallet to take payment

Reply:
That is the most brilliant and terrifying Paul analogy yet 😂